content/en/llm_observability/ragas_evaluations.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Ragas Evaluations
+further_reading:
+    - link: '/llm_observability/submit_evaluations'
+      tag: 'Documentation'
+      text: 'Submit Evaluations'
+---
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">LLM Observability is not available in the selected site ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+[Ragas][1] is an evaluation framework for retrieval augmented generation (RAG) applications. Datadog's Ragas integration enables you to evaluate your production application with faithfulness, answer relevancy, and context precision scores. You can use these scores to find traces that have a high likelihood of inaccurate answers and review them to improve your RAG pipeline.
+
+For a simplified setup guide, see [Ragas Quickstart][7].
+
+<div class="alert alert-warning">
+Datadog recommends that you use sampling for Ragas evaluations. These LLM-as-a-judge evaluations are powered by your LLM provider's account. Evaluations are automatically traced and sent to Datadog. These traces contain LLM spans, which may affect your LLM Observability billing. See <a href="#Sampling">Sampling</a>.
+</div>
+
+## Evaluations
+
+### Faithfulness
+
+The _Faithfulness_ score evaluates how consistent an LLM's generation is against the provided ground truth context data.
+
+This score is generated through three steps:
+1. **Creating statements**: Asking another LLM to break down an answer into individual statements
+2. **Creating verdicts**: For each statement, determining if it is unfaithful to the provided context
+3. **Computing a score**: Dividing the number of contradicting statements over the total number of statements
+
+For more information, see [Ragas' Faithfulness documentation][2].
```

Comment:
```suggestion
For more information, see [Ragas's Faithfulness documentation][2].
```
I would say Ragas's just because I don't think Ragas is intended as a plural entity, and the style guide adds an s in that case ([relevant section of the guide](https://datadoghq.atlassian.net/wiki/spaces/WRITING/pages/3541042054/Style+Guide+Word+List#apostrophe)).